src/pages/App.tsx

```diff
@@ -146,6 +149,15 @@ export default function App() {
     setScrolledState(false)
   }, [pathname])
 
+  const [searchParams] = useSearchParams()
+  useEffect(() => {
+    if (searchParams.get('disableNFTs') === 'true') {
+      setShouldDisableNFTRoutes(true)
```

Comment:
Deriving a state flag from the parameter and using the sessionstorage atom allows us to persist the disabled experience without ensuring that all navigation through the app persists query parameters. So if someone goes to the LP page and creates a new position that has certain min/max tick values and a fee encoded in the query string, and that code doesn't explicitly ensure persistence of existing parameters, then the NFT experience exclusion will cease. 

The ask here is that we create a query parameter that can be linked from a wallet browser that will disable the NFT experience. The most reliable way to do that is to persist it to the user's session instead of auditing existing routing code and then remembering that we need to ensure param persistence in future routing code.

Reply:
yes exactly

I'll add a doc comment to the atom to explain what's going on.

Reply:
Would it make sense to - rather than creating a new atom - extend our existing `featureFlag` mechanism?

TL;DR add a new feature flag for NFT section enabled / disabled and provide a mechanism for setting feature flag values by passing them as a query string - ideally, as a generic solution - but can be made for this one specifically too.

I would do a combination of this hook that checks for query parameters and combine it with `useUpdateFlag`. If generic, I would look into URL encoding a JSON as as `flags` query param. If dedicated, I'd stick to `disableNFTs`.